### DISCUSSION TITLE
Ingestion APIs not working on local setup of cQube

### DISCUSSION THREAD

**Discussion by shubham54-sca** [Sunbird-cQube/community#801] [Q&A]
I have successfully installed the cQube on Linux machine but while perform data ingestion through postman collection provided by cqube team facing below errors.

1. dimension_schema all apis give error "Dimension name is missing".
2. VSK_grammar apis give error "Event name is missing".
3. csv_import_event api give success "file is being processed" but when I checked the logs getting event not found error.

Pls guide if I'm calling APIs in wrong way.

@DevashishRaghav

**Comment by DevashishRaghav** [comment]
@padmanabha1234 please help troubleshoot

**Comment by RNarmatha** [comment]
Please follow the steps :

step_1: After local installations first need to create the jwt token for autherization 
        API Endpoint: http://localhost:3000/generatejwt
        
step_2: After created the JWT token then uploading the VSK schemas files one by one
        API Endpoint: http://localhost:3001/event
        
step_3: After uploaded the VSK schemas then upload the dimension schema one by one
        API Endpoint: http://localhost:3001/dimensions

step_4: After uploaded the schemas then need to upload the vsk raw data(NCERT data) files to mino bucket
        API End point: http://localhost:3000/national_programs  
        
step_5: After uploaded the vsk raw data(NCERT data) then need to upload raw dimension files to minio bucket
        API End point: http://localhost:3000/new_programs  
        

step_6: After uploading VSK data and dimension data need to check file is there in minio or not.if file is present then we can scheduling API's

step_7: After uploading the dimensions then scheduling the API
        API End point: https://cqube-apr27-demo.tibilprojects.com/api/spec/schedule
        body: {
                 "processor_group_name": "Run_adapters", and  "Run Latest Code local"
                 "scheduled_at": "0 21 10 * * ?"
              }
              
step_8: Check the visualization.
[
Document](https://docs.google.com/document/d/1xSTm90t_CsOqbsHc9QDzqhkhATjGzY66lz30F-LKHWo/edit#)
